test/framework/src/main/java/org/elasticsearch/repositories/blobstore/ESBlobStoreRepositoryIntegTestCase.java

```diff
@@ -161,6 +167,66 @@ public void testMultipleSnapshotAndRollback() throws Exception {
         }
     }
 
+    public void testIndicesDeletedFromRepository() throws Exception {
+        Client client = client();
+
+        logger.info("-->  creating repository");
+        final String repoName = "test-repo";
+        assertAcked(client.admin().cluster().preparePutRepository(repoName)
+                        .setType("fs").setSettings(Settings.builder().put("location", randomRepoPath())));
```

Comment:
This seems wrong to me: this class is abstract and contains tests that should work with any type of repository. We should not force the repository type to `fs` but rely on the `createTestRepository(repoName)` method instead like it is done in the other test.